content/en/docs/components/pipelines/sdk/connect-api.md

```diff
@@ -53,7 +53,51 @@ because it requires authentication. Refer to distribution specific documentation
 
 ### Connect to Kubeflow Pipelines from the same cluster
 
-Note, this is not supported right now for multi-user Kubeflow Pipelines, refer to [Multi-User Isolation for Pipelines -- Current Limitations](/docs/components/pipelines/multi-user/#current-limitations).
+#### Multi-User mode
+
+Authentication uses ServiceAccountToken 
+[projection](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection) where token with limited lifetime, that can be verified by Kubeflow Pipelines API, is being injected into (e.g. Jupyter notebook's) Pod.
+
+In order to **access Kubeflow Pipeline from Jupyter notebook**, additional per namespace (profile) manifest is required:
+
+```yaml
+apiVersion: kubeflow.org/v1alpha1
+kind: PodDefault
+metadata:
+  name: access-ml-pipeline
+  namespace: "<YOUR_USER_PROFILE_NAMESPACE>"
+spec:
+  desc: Allow access to Kubeflow Pipelines
+  selector:
+    matchLabels:
+      access-ml-pipeline: "true"
+  volumes:
+    - name: volume-kf-pipeline-token
+      projected:
+        sources:
+          - serviceAccountToken:
+              path: token
+              expirationSeconds: 7200
+              audience: pipelines.kubeflow.org      
```

Comment:
I think it's important to explain how this works, as most users will have never seen this feature of Kubernetes.

Also, we should clarify that this `serviceAccountToken` method respects the Pipelines RBAC, and does not allow access beyond what the `ServiceAcount` running the notebook Pod has.

Reply:
I didn't want to overflow users with details some of them might not find important if their goal is just to have in-cluster access to pipelines working. 
Would adding something like this be enough, or you want to provide more info:
"More details about PodDefault can be found [here](https://github.com/kubeflow/kubeflow/blob/master/components/admission-webhook/README.md)".

Reply:
+1 for the detailed info, because they need the details to understand how to access KFP API from any pod.

I think your argument about focusing on the minimum they need to know is also valid. What about adding a section at the end of the page to introduce the details.

Reply:
> "More details about PodDefault can be found here".

It's should be good enough for me